.github/workflows/pytest_matrix.yml

```
@@ -27,11 +27,7 @@ jobs:
       (github.event_name == 'push' && github.ref == 'refs/heads/main')
     strategy:
       matrix:
-        python-version: [
-            "3.10",
-            "3.11",
-            # "3.12",  # `requests-cache` blocker: https://github.com/airbytehq/airbyte-python-cdk/issues/299
-          ]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Reverting to previous format - allowing each line to be edited or commented-on distinctly, with a clean diff. Wait one minute and then pull the latest.


```suggestion
        python-version: [
            "3.10",
            "3.11",
            "3.12",
            "3.13",
          ]
```